ghost/core/core/server/models/base/plugins/raw-knex.js

```diff
@@ -166,7 +165,7 @@ module.exports = function (Bookshelf) {
                         })();
                     });
 
-                    return Promise.props(props)
+                    return Promise.all(Object.values(props))
```

Comment:
Promise.all returns arrays of resolved promises, which means the original keys are lost. Need to add the original keys back to the resolved promises array from Promise.all.